etc/answers-headless.api.md

```diff
@@ -168,6 +184,16 @@ export function createNearMeFilter(position: NearFilterValue): Filter;
 // @public
 export function createNumberRangeFilter(fieldId: string, range: BoundedRange<number>): FilterTypes;
 
+// @public
+export interface CustomClientSDKHeaderValues extends ClientSDKHeaderValues {
+    'ANSWERS_HEADLESS'?: never;
+}
+
+// @public
+export interface CustomHttpHeaders extends AdditionalHttpHeaders {

```

Comment:
Does this need to extend AdditonalHttpHeaders? It looks like it's overwriting the 'Client-SDK' prop anyway. so I'm not sure what benefit the `extends` gives us

Reply:
At the moment, it isn't necessary. But, I thought it might be better to extend it so that in the future, if more headers are allowed, we don't need to change this to allow them.